sanitizers/src/main/java/com/code_intelligence/jazzer/sanitizers/RegexRoadblocks.java

```diff
@@ -65,13 +65,6 @@ public final class RegexRoadblocks {
   private static final ThreadLocal<WeakHashMap<Object, Character>> PREDICATE_SOLUTIONS =
       ThreadLocal.withInitial(WeakHashMap::new);
 
-  // Do not act on instrumented regexes used by Jazzer internally, e.g. by ClassGraph.
-  private static boolean HOOK_DISABLED = true;
-
-  static {
-    Jazzer.onFuzzTargetReady(() -> HOOK_DISABLED = UNSAFE == null);
```

Comment:
It's part of our public API and very low cost to keep around, so I wouldn't want to delete it just because it's not used by us anymore. What do you think?

Reply:
Let's defer to a follow-up if needed since everything else is resolved.